getting-started.ipynb

```diff
@@ -0,0 +1,601 @@
+      "cell_type": "markdown",
+      "source": [
+        "## Installation\n",
+        "\n",
+        "1.  From `pip`\n",
+        "\n",
+        "``` shell\n",
+        "pip install pytorch-ignite\n",
+        "```\n",
+        "\n",
+        "1.  From `conda`\n",
+        "\n",
+        "``` shell\n",
+        "conda install ignite -c pytorch\n",
+        "```\n",
+        "\n",
+        "See [here](https://pytorch-ignite.netlify.app/docs/how-to-guides/installation) for other installation\n",
+        "options."
+      ],
+      "metadata": {
+        "id": "XTHzzYyoEcFy"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Code\n",
+        "\n",
+        "Import the following:"
+      ],
+      "metadata": {
+        "id": "DcnSr5sGEcFz"
+      }
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 3,
+      "source": [
+        "import torch\n",
+        "from torch import nn\n",
+        "from torch.utils.data import DataLoader\n",
+        "from torchvision.datasets import MNIST\n",
+        "from torchvision.models import resnet18\n",
+        "from torchvision.transforms import Compose, Normalize, ToTensor\n",
+        "from tqdm import tqdm\n",
+        "\n",
+        "from ignite.engine import Engine, Events, create_supervised_trainer, create_supervised_evaluator\n",
+        "from ignite.metrics import Accuracy, Loss"
+      ],
+      "outputs": [],
+      "metadata": {
+        "id": "Saizk3heEcFz"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Speed things up by setting [device](https://pytorch.org/docs/stable/tensor_attributes.html#torch.torch.device) to `cuda` if available else `cpu`."
+      ],
+      "metadata": {
+        "id": "ecMYtJF7OvgT"
+      }
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 8,
+      "source": [
+        "device = torch.device(\"cuda\" if torch.cuda.is_available() else \"cpu\")"
+      ],
+      "outputs": [],
+      "metadata": {
+        "id": "sdjDKcFhOuQn"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Define a class of your model or use the predefined ResNet18 model (modified for MNIST) below, instantiate it and move it to device:"
+      ],
+      "metadata": {
+        "id": "4r_PUH1yEcF1"
+      }
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 5,
+      "source": [
+        "class Net(nn.Module):\n",
+        "\n",
+        "    def __init__(self, in_channels=1):\n",
+        "        super(Net, self).__init__()\n",
+        "        \n",
+        "        # Changed the output layer to output 10 classes instead of 1000 classes\n",
+        "        self.model = resnet18(num_classes=10)\n",
+        "\n",
+        "        # Changed the input layer to take grayscale images for MNIST instaed of RGB images\n",
+        "        self.model.conv1 = nn.Conv2d(1, 64, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3), bias=False)\n",
```

Comment:
Please reduce `kernel_size=(7, 7)` to something smaller, .e.g 3x3

Reply:
Done!